src/api/backend/model/ProfileWorkerModel.js

```diff
@@ -1,13 +1,90 @@
 import PropTypes from "prop-types"
 import AbstractModel from './../../../models/AbstractModel'
+import ProfileWorkerSocialModel from './ProfileWorkerSocialModel'
+import ProfileWorkerServiceModel from './ProfileWorkerServiceModel'
+import ProfileWorkerEmploymentModel from './ProfileWorkerEmploymentModel'
+import CurrencyModel from './CurrencyModel'
+import AttachmentModel from './AttachmentModel'
 
 const schemaFactory = () => ({
+  id: PropTypes.string,
+  approved: PropTypes.bool,
   isRequested: PropTypes.bool,
+  regular: PropTypes.shape({
+    currencies: PropTypes.arrayOf(PropTypes.instanceOf(CurrencyModel)),
+    hourlyCharge: PropTypes.string,
+  }),
+  verifiable: PropTypes.shape({
+    intro: PropTypes.string,
+    pageBackground: PropTypes.string,
+    attachments: PropTypes.arrayOf(PropTypes.instanceOf(AttachmentModel)),
+  }),
+  custom: PropTypes.any,
+  socials: PropTypes.arrayOf(PropTypes.instanceOf(ProfileWorkerSocialModel)),
+  services: PropTypes.arrayOf(PropTypes.instanceOf(ProfileWorkerServiceModel)),
+  employments: PropTypes.arrayOf(PropTypes.instanceOf(ProfileWorkerEmploymentModel)),
```

Comment:
this is  not correct schema
see https://github.com/ChronoBank/LaborX-backend/blob/develop/lib/models/Profile/lib/ProfileWorkerModel.js
client model must be same